lib/components/PlayerScreen/player_buttons.dart

```diff
@@ -42,10 +41,11 @@ class PlayerButtons extends StatelessWidget {
                     : null,
               ),
               _RoundedIconButton(
-                width: targetHeight > 264 ? 62 : 48,
-                height: targetHeight > 264 ? 62 : 48,
-                borderRadius:
-                    BorderRadius.circular(targetHeight > 264 ? 16 : 12),
+                width: controller.shown(PlayerHideable.bigPlayButton) ? 62 : 48,
+                height:
+                    controller.shown(PlayerHideable.bigPlayButton) ? 62 : 48,
+                borderRadius: BorderRadius.circular(
+                    controller.shown(PlayerHideable.bigPlayButton) ? 16 : 12),
```

Comment:
At this point, maybe we should just create two instances of `_RoundedIconButton`, one called `SmallPlayButton` and the other `BigPlayButton`, that construct a `_RoundedIconButton` with the appropriate size and border radius?

Reply:
Maybe, but it doesn't seem worth the effort of changing now without stronger reasoning.

Reply:
Yeah it doesn't have to be part of this PR, or maybe I'll just add it myself ^^